### PR TITLE
reduced lib size from 3,009,838 -> 1,956,940

### PR DESCRIPTION
**Pull request by jay-dee7**
Signed-off-by: Jasdeep Singh <jasdeepsingh.uppal@gmail.com>

### CODE REVIEW

**Comment by jay-dee7** [comment]
@Arachnid this improves on #26, size is reduced further but i think it needs more time

File: src/index.ts [hunk 1]
```diff
@@ -1,20 +1,20 @@
-// tslint:disable-next-line:no-var-requires
-import { decodeAddress as polkadotDecodeAddress, encodeAddress as polkadotEncodeAddress }  from '@polkadot/util-crypto/address';
+import {decodeAddress, encodeAddress} from '@polkadot/util-crypto/address'
```
Comment (Arachnid) [inline src/index.ts]: Can you please give these more specific names?

File: src/index.ts [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 import { decode as bech32Decode, encode as bech32Encode, fromWords as bech32FromWords, toWords as bech32ToWords } from 'bech32';
 import { decode as bs58checkDecode, encode as bs58checkEncode } from 'bs58check';
+// tslint:disable-next-line:no-var-requires
 import { decode as cashaddrDecode, encode as cashaddrEncode } from 'cashaddrjs';
-import { PublicKey as eosPublicKey } from 'eosjs-ecc';
 // @ts-ignore
-import { b32decode, b32encode, isValid   } from 'nem-sdk/build/model/address'
+import PublicKey from 'eosjs-ecc/lib/key_public';
```
Comment (Arachnid) [inline src/index.ts]: Likewise, can you give this a more specific name?

File: src/index.ts [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 // @ts-ignore
-import { hex2a, ua2hex  } from 'nem-sdk/build/utils/convert.js';
-import * as ripple from 'ripple-address-codec';
+import { b32decode, b32encode, isValid   } from 'nem-sdk/build/model/address';
+// @ts-ignore
+import { hex2a, ua2hex  } from 'nem-sdk/build/utils/convert';
+import { codec } from 'ripple-address-codec/dist/xrp-codec';
```
Comment (Arachnid) [inline src/index.ts]: Likewise, can you give this a more specific name?

File: src/index.ts [hunk 4]
```diff
@@ -234,25 +234,28 @@ function b32decodeXemAddr(data: string): Buffer {
...
     throw Error('Unrecognised address format');
   }
-  return eosPublicKey(data).toBuffer();
+  return PublicKey(data).toBuffer();
 }
 
 function ksmAddrEncoder(data: Buffer): string {
-  return polkadotEncodeAddress(data, 2);
+  return encodeAddress(data, 2)
+  // return encodeAddress(data, 2);
```
Comment (Arachnid) [inline src/index.ts]: Please delete any commented lines that aren't needed.

**Comment by jay-dee7** [comment]
@Arachnid i've updated this PR with requested changes, also, i've added size checking as part of travis and as of this PR, lib size sits at **1956940**

**Pull request merged**